index.bs

```diff
@@ -2375,12 +2402,64 @@ data-frame = {
   ? 5: media-time ; sync-time
 }
 
+; TODO(jophba): Why do some have type keys, but not others?
 ; type key 120
 video-frame-request = {
  1: uint; encoding-id
- 2: uint; sequence-number: 
+ 2: uint; sequence-number
  ? 3: uint; highest-decoded-frame-sequence-number
 }
 
+streaming-capabilities-request = {
+  request
+}
+
+streaming-capabilities-response = {
+  response
+  1: streaming-capabilities ; streaming-capabilities
+}
+
+streaming-capabilities = {
+  0: [* receive-audio-capability] ; receive-audio
+  1: [* receive-video-capability] ; receive-video
+  2: [* receive-data-capability] ; receive-data
+}
+
+codec = {
```

Comment:
We could just make this a "format" as the type and have the field name for audio and video be "codec" so ".codec.name" and ".codec.parameters" works

Reply:
Sounds good to me, I went back and forth on this.